src/utils/index_html.cr

```diff
@@ -56,6 +56,22 @@ module Mint
                   href: path_for("icon-#{size}x#{size}.png"))
               end
             end
+
+            unless json.external_files["css"].empty?
+              json.external_files["css"].each do |stylesheet|
+                # Assuming File Is In `public` Directory
```

Comment:
We don't need to assume because when parsing the `mint.json` we make sure that the file exists relative to the root directory (where the `mint.json` is).

There is two ways we can handle this:
- either we force the assets to be in the public directory which is reasonable, if so we need to check that when we parse them
- we use directory name which won't conflict with user paths, I am thinking about `.mint/css` and use that in both development and production, but then we need to implement the serving in the `Reactor`

I think probably forcing them to put assets like these in the public directory would be preferable, this would apply to the `javascripts` as well)

Reply:
Also, should we provide the option to link the javascripts as `script.src` vs being baked in with the application? In order to take advantage of browser caching, parallel execution, etc...

Reply:
We could also add a asset loader, which will load external `javascripts`.

Reply:
What I'm trying to convey is that Code Splitting is useful if we care about application start-up time, faster repeat load times, etc,

Reply:
I think service them as `<script src=""></script>` would be better, it can be an done in an other issue if like.